core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -236,13 +236,19 @@ class GroupCoordinator(val brokerId: Int,
                       generation: Int,
                       memberId: String,
                       groupAssignment: Map[String, Array[Byte]],
-                      responseCallback: SyncCallback) {
-    if (!isActive.get) {
-      responseCallback(Array.empty, Errors.COORDINATOR_NOT_AVAILABLE)
-    } else if (!isCoordinatorForGroup(groupId)) {
-      responseCallback(Array.empty, Errors.NOT_COORDINATOR)
-    } else {
-      groupManager.getGroup(groupId) match {
+                      responseCallback: SyncCallback): Unit = {
+    validateGroup(groupId) match {
+      case Some(error) if error == Errors.COORDINATOR_LOAD_IN_PROGRESS =>
```

Comment:
I wanted to do that, but it would break older clients. Maybe we can leave a note somewhere to do this in the next protocol bump? I guess we could leave out the client handling of the error until then. What do you think?